### ISSUE TITLE
Set kclr terminfo value

### ISSUE THREAD

**Issue by Koxiaet** [alacritty/alacritty#4500]
For me Neovim doesn't clear the screen on exit in Alacritty unless I set `kclr=\E[3;5~` in its `terminfo`. I don't know why this is the case, I just found it out by trial and error. It also appears that `\E[3;5~` is the same sequence declared in the nonstandard Terminfo property `kDC5` that Alacritty sets for some reason.

**Comment by chrisduerr** [comment]
What do you mean by "clear the screen"? Is neovim not running in the alternate screen buffer for you?

**Comment by Koxiaet** [comment]
Nevermind, I was entirely mistaken, sorry for the inconvenience.

Should anybody stumble upon this issue in the future: this was all due to Neovim looking at `$TERMINFO` before `$TERMINFO_DIRS`. I had `$TERMINFO` set to `~/.local/share/terminfo`, which unbeknownst to me contained my previous terminal. When I realized Neovim wasn't working on Alacritty I infocmp'd the two terminals, and made the changes to `/usr/share/terminfo/a/alacritty` - but it wasn't the change that fixed it, it was me accidentally running `tic` before I set `$TERMINFO` to `/usr/share/terminfo`. Then, when I went to test my hypothesis that it was `kclr` I removed `kclr` from `/usr/share/terminfo/a/alacritty` and removed `~/.local/share/terminfo/a/alacritty`. Of course it didn't work, but what I didn't realize was that it was removing `~/.local/share/terminfo/a/alacritty` that made it fail and not removing `kclr`.
